_data/team/roles/code_of_conduct.yml

```diff
@@ -0,0 +1,17 @@
+role: Code of conduct committee
+description: |
+  The [code of conduct committee](https://www.mdanalysis.org/pages/conduct/)
```

Comment:
Regrettably no, I couldn't get that working in the 10-15 min I spent looking around Liquid/Jekyll docs and trial-and-erroring -- maybe someone who knows more about this has the solution, but for me it would just render as the raw `{{ site.baseurl }}{% link pages/conduct.md %}` link.

Reply:
Thanks for looking into it, much appreciated!